src/Script/RunningScript.ts

```diff
@@ -58,6 +58,9 @@ export class RunningScript {
   // Process ID. Must be an integer and equals the PID of corresponding WorkerScript
   pid = -1;
 
+  // Process ID of the parent process. Defaults to 0.
```

Comment:
```suggestion
  // Process ID of the parent process. 0 indicates no parent (such as run from terminal).
```
It is self-evident that the default is 0. XD